youtube2mediawiki.py

```diff
@@ -519,8 +520,32 @@ def safe_name(s):
     s = s.replace('__', '_').replace('__', '_')
     return s
 
+def ffmpeg_installed():
+    if (platform.system() == 'Linux' or platform.system() == 'Darwin'):
+        cmd_path = 'which'
+        ffmpeg = 'ffmpeg'
+    else: #platform.system() == 'Windows'
+        cmd_path = 'where'
+        ffmpeg = 'ffmpeg.exe'
+
+    if DEBUG:
+        print 'Testing for ' + ffmpeg + ' on ' + platform.system()
+    if 0 != subprocess.call([cmd_path, ffmpeg], stdout=open(os.devnull, 'wb'), stderr=subprocess.STDOUT):
+        ffmpeg = './' + ffmpeg
+        if not os.path.isfile(ffmpeg):
+            print 'Install ffmpeg or place ' + ffmpeg + ' in the current working directory (' + os.getcwd() + ')'
+            return False
+
+    if 0 == subprocess.call([ffmpeg, '-version'], stdout=open(os.devnull, 'wb'), stderr=subprocess.STDOUT):
+        return ffmpeg
+    else:
+        return False
+
 def import_youtube(youtube_id, username, password, mediawiki_url, name=''):
     yt = Youtube()
+    ffmpeg = ffmpeg_installed()
```

Comment:
this should only check for ffmpeg if MERGE_DASH is true. printing a warning that ffmpeg is not installed if its not needed seams wrong.